### ISSUE TITLE
Feature request- import comments per domain

### ISSUE THREAD

**Issue by p1r473** [jacklul/pihole-updatelists#27]
Hi Jack,
I had a great idea that would make a cool feature request. Let me know what you think.

I have 2 files- one is whitelist.txt and one is whitelist-explanations.txt.
I do the same thing for the blacklist- 2 separate files.
In the first file, is just the domains

whitelist.txt
```
ow.ly
```
In the first file is the explanation of why I am whitelisting it, for future me to know why I whitelisted or blacklisted something

whitelist-explanations.txt
```
ow.ly - URL shortener service
```

I had an idea that if you thought was cool you could implement
When you parse the blacklist/whitelist/adlist, parse the line for a comment. If there is a comment, put that comment into the database

So what I would end up having would be just 1 file which has both the domain AND the comment/explanation:

whitelist.txt
```
ow.ly #URL shortener service
```

Basically, embedding comments for the database directly into whitelist/blacklist/adlists

Just an idea :)

**Comment by jacklul** [comment]
Implementing this could end up with undesired side effects with other remote lists which are also commenting entries.

**Comment by p1r473** [comment]
OK no problem, just an idea, thanks :)

**Comment by jacklul** [comment]
Implemented.

**Comment by p1r473** [comment]
Thanks so much! Love it! If you paste a donation link I will gladly donate for this great feature.

One glitch: Where does these | characters come from? Possible to remove them?
![image](https://user-images.githubusercontent.com/9235633/86520033-9eb98780-be0e-11ea-93d7-a98c9e921612.png)

**Comment by jacklul** [comment]
They are separating comment from the list and the COMMENT value, did you set it to 3 spaces?

**Comment by p1r473** [comment]
Thanks, you fixed it. I was using 3 spaces.
```
; Comment string used to know which entries were created by the script
COMMENT="   "
```
